### PR TITLE
Document the new features of the Cadence testing framework

### PR DESCRIPTION
**Pull request by m-Peter**

### CODE REVIEW

**Review by SupunS — APPROVED** [review]
Thank you for documenting all these!

File: docs/cadence/testing-framework.mdx [hunk 1]
```diff
@@ -263,16 +738,31 @@ access(all) struct interface BlockchainBackend {
...
 }
+
 ```
 
 ### Creating a blockchain
 
-A new blockchain instance can be created using the `newEmulatorBlockchain` method.
-It returns a `Blockchain` which is backed by a new [Flow Emulator](../tools/toolchains/emulator) instance.
+A new blockchain instance can be created using the `Test.newEmulatorBlockchain` method.
+It returns a `Blockchain` which is backed by a new [Flow Emulator](../tooling/emulator) instance.
```
Comment (SupunS) [inline docs/cadence/testing-framework.mdx:760]: I think this link here is causing the CI failure. Can you maybe double-check?

Comment (m-Peter) [reply docs/cadence/testing-framework.mdx]: My bad, I updated my fork, but forgot to pull the changes. I did so when some conflicts showed up, but this one probably sneaked in :sweat_smile:

**Comment by m-Peter** [comment]
The CI seems to have been fixed with Bastian's suggestion :tada:

**Pull request merged**